packages/flutter_tools/lib/src/android/android_studio.dart

```diff
@@ -27,20 +27,38 @@ AndroidStudio get androidStudio => context[AndroidStudio];
 final RegExp _dotHomeStudioVersionMatcher =
     RegExp(r'^\.(AndroidStudio[^\d]*)([\d.]+)');
 final RegExp _pathsSelectorMatcher =
-    RegExp(r'"idea.paths.selector" = "AndroidStudio[^;]+"');
+    RegExp(r'"idea.paths.selector" = "[^;]+"');
```

Comment:
This x = y format is specific to macs and plists right? Add a comment above this and the next variable.